### DISCUSSION TITLE
Italian translations

### DISCUSSION THREAD

**Discussion by Sneffel** [mawoka-myblock/ClassQuiz#220] [General]
The Italian translation hasn't been updated in 6 months.
I've translated 56 phrases based on my logs.
There are a lot of anonymous suggestions from 4 months ago.
It's not uncommon for Italian translations to be completely wrong or missing the whole context.
I guess it's due to the Albanian studying the language abroad.
I'm taking a break at the moment. I'll come later.

**Comment by mawoka-myblock** [comment]
The anonymous suggestions may be from the machine translation
